bimmer_connected/state.py

```diff
@@ -87,17 +87,8 @@ def update_data(self) -> None:
         _LOGGER.debug('requesting new data from connected drive')
 
         response = self._account.send_request(
-            VEHICLE_STATE_URL.format(server=self._account.server_url, vin=self._vehicle.vin))
-
-        attributes = response.json()['attributesMap']
-        if attributes['head_unit'] not in ('NBTEvo', 'EntryEvo', 'NBT', 'EntryNav'):
```

Comment:
can we add that check somewhere else? is there a `head_unit` attribute?

Reply:
Sorry, there is not such attribute. There is however a "hmiVersion" attribute. Not sure how to map that to the head_unit types...

Reply:
Maybe that `hmiVersion` is even better, see https://github.com/ChristianKuehnel/bimmer_connected/pull/43#issuecomment-372069868.